recipes/pycoalescence/conda_build_config.yaml

```diff
@@ -0,0 +1,12 @@
+boost:
+  - 1.67
+python:
+  - 2.7  # [not win]
+  - 3.5  # [not osx]
+  - 3.6
+  - 3.7  # [not win]
+vc:
+  - 14  # [win]
```

Comment:
I'm trying to understand that.
So.. as you are selecting ``vc 14`` it will not create the package for ``python 2.7`` in windows
Because of that windows will be just python 3.5 and 3.6
I think we can try to build that for all python 3

Reply:
I will try one thing